infer_segmentation.py

```diff
@@ -68,31 +97,35 @@ def overlay_predictions(prepared_tiles, preds, prediction_threshold, background_
             prediction_tiles[i].append(np.dstack((prepared_tiles[i][j], prepared_tiles[i][j], prepared_tiles[i][j])))
             prediction_tiles[i][j] = (prediction_tiles[i][j] * 255).astype(int)
 
-            above_threshold_mask = preds[i][j].max(axis=2) > prediction_threshold
-            best_class_by_pixel = preds[i][j].argmax(axis=2)
+            relative_above_threshold_mask = np.divide(preds[i][j], np.multiply(np.ones_like(preds[i][j]),
+                                                                               prediction_threshold)).max(axis=-1) > 1
+            best_class_by_pixel = np.divide(preds[i][j], np.multiply(np.ones_like(preds[i][j]),
+                                                                     prediction_threshold)).argmax(axis=-1)
             color_counter = 0
             for class_i in range(preds[i][j].shape[-1]):
-                above_threshold_and_best_class = above_threshold_mask & (best_class_by_pixel == class_i)
+                rel_above_threshold_and_best_class = relative_above_threshold_mask & (best_class_by_pixel == class_i)
```

Comment:
@Josh-Joseph these lines appear to work fine, but can you look em over to verify things are good from a syntax perspective?